right/src/ledmap.c

```diff
@@ -157,70 +285,133 @@ static void setPerKeyRGB(const rgb_t* color, uint8_t slotId, uint8_t keyId)
     LedDriverValues[slotId][ledMapItem->blue] = color->blue * KeyBacklightBrightness / 255;
 }
 
+static void setPerKeyMonochromatic(const rgb_t* color, uint8_t slotId, uint8_t keyId)
+{
+    const rgb_t *ledMapItem = &LedMap[slotId][keyId];
+    if (ledMapItem->red == 0 && keyId != 0) {
+        return;
+    }
+    uint8_t value = ((uint16_t)color->red + color->green + 2*color->blue) / 4;
```

Comment:
@kareltucek Why double the blue?

Reply:
It is cheaper to compute, as this can be computed just using binary shifts without any divisions...

Reply:
I see, but the blue gets disproportionally weighted, isn't it?

Reply:
Yes, but that does not seem as a big deal, as in practice, I can hardly distinguish lighter keys from darker, let alone trying to guess the original color...

Reply:
I'm all for optimizing the code, but not at the expense of accurate backlight, even if it's barely visible.

Reply:
All right...